assets/js/curlparser.js

```diff
@@ -7,7 +7,7 @@ import * as querystring from "querystring";
  * output this: 'msg1=value1&msg2=value2'
  * @param dataArguments
  */
-function joinDataArguments(dataArguments) {
+ let joinDataArguments=(dataArguments)=> {
```

Comment:
Can you include a space both either side of the function name and equals sign, and the closing parameter bracket and opening function brace:

**Before:**
```javascript
let joinDataArguments=(dataArguments)=> {
```

**After:**
```javascript
let joinDataArguments = (dataArguments) => {
```